### PR TITLE
AM-5785: update the steps in the demo using kubeslice-cli doc

### PR DESCRIPTION
**Pull request by uma-kt**
- Updated the steps.

### CODE REVIEW

File: versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx:83]: This sentence is incomplete: `After successfully installing KubeSlice Controller and the Slice Operator on the worker clusters` - and then what?.  

You must combine the second sentence with this one if the sentence is not too long.

File: versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx:93]: If this is the demo topic, then the demo (minimal | full) both would create a slice right? 

@uma-kt / @priyank-upadhyay  - do we need to describe create a slice topic here or can we skip describing creating a slice and move to onboarding namespaces?

Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx:93]: @veenamj-avesha  yes minimal and full demo creates a slice. Creating a slice is required if the user wants to try creating a new slice.   As per Eric's comment in the group, we can keep that step if the user wants to create a slice.

Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx:93]: I will add a paragraph or two lines explaining the difference between full and minimal demo.

File: versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx
Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx:83]: updated

File: versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx
Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx:93]: added the below note:
:::info
Skips this step if you do not want to create a new slice and continue onboarding the application on the `demo` slice.
:::

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:42]: I think we need to write MAC as `MacOS` to be more clear. 

I remember @priyank-upadhyay  asking us to refer to MacOS.

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:43]: This sentence needs a bit rephrasing. 

You are unable to install because of the error is the scenario. 

due to is usually used when we describe the issue like no disk space or low memory etc. 

So, can you rephrase the above sentence as 'When you try to install kubeslice-cli on MacOS, if you get the `Unverified Developer Error Message`, then it means that you as a developer is not registered with Apple.` 

or something like that... 

Here the developer who installs gets the error, and we describe in second person; so the developer is you... Does it make sense?

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:44]: This sentence is redundant if you add this info in the above sentence ....

**Review by veenamj-avesha — COMMENTED** [review]
Left 2 comments. Please incorporate the feedback.

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:43]: Updated the below sentence:
When you try to install kubeslice-cli on macOS, you get the `Unverified Developer Error Message`.
This error message appears when you try to install an application from a developer who has not registered with Apple.

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:44]: ... when you try to install an application from a developer who has not been registered with Apple  or .. who is not registered with Apple.

**Review by veenamj-avesha — COMMENTED** [review]
Left a comment.

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx]: updated

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:42]: change message to `Message` as other words are in title case.

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:48]: Can you change MAC to macOS?

**Review by veenamj-avesha — APPROVED** [review]
Two more comments - change them before merging this PR. 

I'm approving now as I have reviewed this in multiple iterations.

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:48]: updated

File: versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx
Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx:42]: updated

**Pull request merged**
